scripts/actions/spells/songs/pining_nocturne.lua

```diff
@@ -0,0 +1,151 @@
+
+-- Copied from /scripts/globals/magic.lua so it's duplicative, but it's local to that file.
+-- Can maybe use xi.combat.element.strongAffinityAcc from /scripts/globals/combat/element_tables.lua
+local strongAffinityAcc =
+{
+    xi.mod.FIRE_AFFINITY_ACC,
+    xi.mod.ICE_AFFINITY_ACC,
+    xi.mod.WIND_AFFINITY_ACC,
+    xi.mod.EARTH_AFFINITY_ACC,
+    xi.mod.THUNDER_AFFINITY_ACC,
+    xi.mod.WATER_AFFINITY_ACC,
+    xi.mod.LIGHT_AFFINITY_ACC,
+    xi.mod.DARK_AFFINITY_ACC,
+}
+
+spellObject.onMagicCastingCheck = function(caster, target, spell)
+    return 0
+end
+
+spellObject.onSpellCast = function(caster, target, spell)
+    -- https://www.bg-wiki.com/ffxi/Category:Nocturne
+    -- TODO: Current state may be a precursor to general BRD song handling.
+    -- TODO: Enhancing and Enfeeble songs should have their own separate handling (see below comment for details).
+    -- TODO: https://github.com/LandSandBoat/server/pull/6677#issuecomment-2580655974
+
+    -- Spell parameters
+    local songEffect      = xi.effect.NOCTURNE
+    local power           = 15
+    local powerCap        = 25
+    local powerBoostMult  = 1.5
+    local soulVoiceType   = soulVoiceBoostType.POWER
+    local duration        = 120     -- This is the base duration.
+    local resistThreshold = 0.5
+    local modifierStat    = 0       -- Song+ modifier.
+    local augmentStat     = 0       -- Most songs only augment their own effect, but others (like Etudes) augment different stats.
+    local jobPointEffect  = 0
+    local dotTick         = 0
+
+    -- Check to see if the player has any elemental affinity gear like an elemental staff.
+    local elementalAccBonus = caster:getMod(strongAffinityAcc[spell:getElement()]) * 10
+
+    -- Check the amount of Song+ and All_Song+ gear.
+    local gearBoost = caster:getMod(modifierStat) + caster:getMod(xi.mod.ALL_SONGS_EFFECT)
+
+    -- BG Wiki doesn't say explicitly that Song+ gear affects accuracy for threnodies.
+    if songEffect ~= xi.effect.THRENODY then
+        elementalAccBonus = elementalAccBonus + gearBoost
+    end
+
+    -- TODO: Resistance can be modularlized.
+    -- TODO: Add handling for job point song accuracy bonus (xi.jp.SONG_ACC_BONUS)
+    -- TODO: Add handling for job point gifts for magic accuracy. (5 @ 60, 8 @ 360, 10 @ 910, 13 @ 1710)
+    -- Check for resistance.
+    local resistParams = {}
+    resistParams.attribute = xi.mod.CHR
+    resistParams.skillType = xi.skill.SINGING
+    resistParams.bonus     = elementalAccBonus
+    if applyResistanceEffect(caster, target, spell, resistParams) < resistThreshold then
+        spell:setMsg(xi.msg.basic.MAGIC_RESIST)
+        return songEffect
+    end
+
+    -- TODO: Power calculation can be modularized.
+    power = math.min(power, powerCap)               -- Cap power if necessary. (Applied before Merits, Job-Points and Status-Effects)
+    power = power + (gearBoost * powerBoostMult)    -- Boost from Song+ and All_Song+ gear.
```

Comment:
This applies to other lines, not just this one:
`power = power + gearBoost * powerBoostMult`

Unnecesary parenthesis are unnecesary.